en/components/avatar.md

```diff
@@ -12,6 +12,16 @@ Use the Avatar Component as a graphical representation of a person through a pro
 
 <img class="responsive-img" src="../images/avatar_demo.png" srcset="../images/avatar_demo@2x.png 2x" />
 
+## Type
+
+The Avatar can carry different types of content such as an **image**, a string with **initials**, or an **icon**.
+
+<img class="responsive-img" src="../images/avatar_content.png" srcset="../images/avatar_content@2x.png 2x" />
+
```

Comment:
Once you accept the Shape heading move the section for shape after the one for size.